### ISSUE TITLE
Tree with support

### ISSUE THREAD

**Issue by DiedeMaas** [fgvieira/ngsDist#4]
Hi,

I am trying to compute a Tree via FigTree using pairwise genetic distances computed by ngsDist and as explained by this tutorial. I have followed the steps in performing bootstraps because I want to show the support on the tree.

However, when I try the command in RAxML as stated in the tutorial it gives me an error because -f i is not in the program. I've tried some other options for -f but it always asks me for to give an alignment file via the -s argument. However, I do not have an alignment file, and from the tutorial it does not seem like you need one.

raxmlHPC -f i -t Brach_boot_main.tree -z Brach_boot_boot.tree -m GTRCAT -n Brach_boot

In short: I would like to display support values on my tree in FigTree, but cannot figure out how I should do that. I am able to make the trees themselves, just not with support values.

Thank you,
Diede

**Comment by fgvieira** [comment]
Hi Diede,

what version of RAxML are you using? Try updating to the latest version...
If you need more info, you can check RAxML [manual](http://sco.h-its.org/exelixis/resource/download/NewManual.pdf).

Best,

**Comment by DiedeMaas** [comment]
Hi Filipe,

It was indeed a problem with the version of RAxML, sorry to have bothered you with such a trivial thing! But thanks for your swift response. :)

Diede
